test/test_cuda.py

```diff
@@ -1136,6 +1136,7 @@ def test_scatter_cpu_sizes(self):
     def test_scatter_gpu(self):
         self._test_scatter(torch.randn(4, 4).cuda(), dim=0)
 
+    @skipIfRocm
```

Comment:
Does this test before this PR?

Reply:
No, this came in broken with a recent upstream merge and is skipped here to fix CI tests.

Reply:
But we don't see it failing in current pytorch master

Reply:
By upstream you mean pytorch master or your fork?

Reply:
upstream == pytorch master

it certainly failed for us on CI, maybe it's just flaky. We have a few that behave like that and in the interest of stability I disable them.

Reply:
AFAICT there is no flakiness has been observed in the pass couple days in the upstream CI. Are you sure it's not some changes done in the fork that cause the failure?